js/bybit.js

```diff
@@ -2458,284 +2548,328 @@ module.exports = class bybit extends Exchange {
         }
-        const clientOrderId = this.safeString2 (params, 'order_link_id', 'clientOrderId');
-        if (clientOrderId !== undefined) {
-            request['order_link_id'] = clientOrderId;
-            params = this.omit (params, [ 'order_link_id', 'clientOrderId' ]);
+        if (price !== undefined) {
+            request['price'] = price;
         }
         const stopPx = this.safeValue2 (params, 'stop_px', 'stopPrice');
-        const basePrice = this.safeValue (params, 'base_price');
-        let method = undefined;
-        if (market['swap']) {
-            if (market['linear']) {
-                method = 'privateLinearPostOrderCreate';
-                request['reduce_only'] = false;
-                request['close_on_trigger'] = false;
-            } else if (market['inverse']) {
-                method = 'v2PrivatePostOrderCreate';
-            }
-        } else if (market['future']) {
-            method = 'futuresPrivatePostOrderCreate';
-        }
+        const basePrice = this.safeValue2 (params, 'base_price', 'basePrice');
+        let isConditionalOrder = false;
         if (stopPx !== undefined) {
+            isConditionalOrder = true;
             if (basePrice === undefined) {
                 throw new ArgumentsRequired (this.id + ' createOrder() requires both the stop_px and base_price params for a conditional ' + type + ' order');
-            } else {
-                if (market['swap']) {
-                    if (market['linear']) {
-                        method = 'privateLinearPostStopOrderCreate';
-                    } else if (market['inverse']) {
-                        method = 'v2PrivatePostStopOrderCreate';
-                    }
-                } else if (market['future']) {
-                    method = 'futuresPrivatePostStopOrderCreate';
-                }
-                request['stop_px'] = parseFloat (this.priceToPrecision (symbol, stopPx));
-                request['base_price'] = parseFloat (this.priceToPrecision (symbol, basePrice));
-                request['trigger_by'] = 'LastPrice';
-                params = this.omit (params, [ 'stop_px', 'stopPrice', 'base_price', 'trigger_by' ]);
             }
-        } else if (basePrice !== undefined) {
-            throw new ArgumentsRequired (this.id + ' createOrder() requires both the stop_px and base_price params for a conditional ' + type + ' order');
+            request['stop_px'] = parseFloat (this.priceToPrecision (symbol, stopPx));
+            request['base_price'] = parseFloat (this.priceToPrecision (symbol, basePrice, 'basePrice'));
+            const triggerBy = this.safeString2 (params, 'trigger_by', 'triggerBy', 'LastPrice');
+            request['trigger_by'] = triggerBy;
+            params = this.omit (params, [ 'stop_px', 'stopPrice', 'base_price', 'triggerBy', 'trigger_by' ]);
+        }
+        const clientOrderId = this.safeString2 (params, 'clientOrderId', 'orderLinkId');
```

Comment:
Houston, we have a problem here, lost the correct `order_link_id` field: https://bybit-exchange.github.io/docs/linear/#t-placeactive
I would appreciate if you could restore the commented fields in the `request` cause that helps align them with the docs.

Reply:
@kroitor I'll